vms/platformvm/txs/executor/reward_validator_test.go

```diff
@@ -137,7 +138,7 @@ func TestRewardValidatorTxExecuteOnCommit(t *testing.T) {
 
 	onCommitBalance, err := avax.GetBalance(env.state, stakeOwners)
 	require.NoError(err)
-	require.Equal(oldBalance+stakerToRemove.Weight+27697, onCommitBalance)
+	require.Equal(oldBalance+stakerToRemove.Weight+38944, onCommitBalance)
```

Comment:
This is a terrible test. But out-of-scope for this PR.

Reply:
This magic number is signifying the staking rewards that the genesis validator will receive. The reason this number changes is because the genesis staking duration was increased from `20` days to `28` days (because `28` days was used in most other places)